extensions/azurePublishNew/src/node/azureResources/constants.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
I moved these constants into a constants file as they are used across the resources.  There are some grouping improvements we can do later rather than having a set of individual strings.